tools/image_processing/bia-ftplinks/ftpLink.txt

```diff
@@ -0,0 +1,2 @@
+null
```

Comment:
is this correct?

Reply:
ah, it shouldn't be there. I removed it.